core/js/src/main/scala/perfolation/DoubleImplicits.scala

```diff
@@ -0,0 +1,17 @@
+package perfolation
+
+import java.math.RoundingMode
+import java.util.Currency
+
+class DoubleImplicits(val d: Double) extends AnyVal {
+  def f(i: Int = 1,
+        f: Int = 2,
+        maxI: Int = 9,
+        maxF: Int = 100,
+        g: Boolean = true,
+        c: Option[Currency] = None,
+        rm: RoundingMode = RoundingMode.HALF_UP): String = {
+    d.toString
+//    ThreadLocalNumberFormat(i, f, maxI, maxF, g, c, rm).format(d)
```

Comment:
I hope you realize I can't merge this until this is fixed, right?

Reply:
Of course not :) It was more of an experiment to see what the difference in size is. For us, the problem is that we do not really need such sophisticated formatters and it is totally not worth 600 kB in our javascript asset. Logging in the frontend is mostly just for development, in production we only log exceptional problems to our backend.

Alternatively, could we make this formatting somehow optional? Maybe in a separate subproject in scribe that I can depend on?

Reply:
Perhaps, but I think we should just figure out a pure Scala way to accomplish this without using Java classes at all and everyone will be happy.